src/css/main.less

```diff
@@ -966,24 +967,33 @@ fieldset[disabled] .btn-info.active {
       padding: @base-spacing-unit*2;
       width: @sidebar-width;
 
-      .btn-success {
+      .btn-success, .dropdown-toggle, .dropdown-menu {
```

Comment:
If this many classes have this same style, does it makes sense to introduce a new class for this and use this new class in the markup classNames? Only an idea..